ivi-layermanagement-examples/simple-weston-client/src/simple-weston-client.c

```diff
@@ -887,24 +887,8 @@ int main (int argc, const char * argv[])
 
 Error:
 #ifdef LIBWESTON_DEBUG_PROTOCOL
-    weston_debug_v1_destroy(wlcontext->debug_iface);
-
-    while (1) {
-        struct debug_stream *stream;
-        int empty = 1;
-
-        wl_list_for_each(stream, &wlcontext->stream_list, link)
-            if (stream->obj) {
```

Comment:
If we run the example with streams are enabled by the IVI_CLIENT_DEBUG_STREAM_NAMES  variable, the stream->obj  will valid and it does not destroy until the weston exit or destroy_streams is called. So, we will get the loop for ever here, if weston doesn't stop.